spec/system/courses/students_spec.rb

```diff
@@ -14,21 +14,35 @@
   let(:team_coach) { create :faculty, school: school }
 
   # Create few teams
-  let!(:team_1) { create :startup, level: level_1, name: 'Zucchini', tag_list: ['starts with z', 'vegetable'] } # This will always be around the bottom of the list.
-  let!(:team_2) { create :startup, level: level_2, name: 'Asparagus', tag_list: ['vegetable'] } # This will always be around the top.
+  let!(:team_1) do
+    create :startup,
+           level: level_1,
+           name: 'Zucchini',
+           tag_list: ['starts with z', 'vegetable']
+  end # This will always be around the bottom of the list.
+  let!(:team_2) do
+    create :startup, level: level_2, name: 'Asparagus', tag_list: ['vegetable']
+  end # This will always be around the top.
   let!(:team_3) { create :startup, level: level_2, name: 'Banana' }
   let!(:team_4) { create :startup, level: level_3, name: 'Blueberry' }
   let!(:team_5) { create :startup, level: level_3, name: 'Cherry' }
   let!(:team_6) { create :startup, level: level_3, name: 'Elderberry' }
+  let(:first_student) { team_1.founders.first.user }
```

Comment:
This `let` appears to be superfluous. It's used only once on L34... might as well write `team_1.founders.first.user.update!(last_seen_at: 3.days.ago)` there. 🤔